deltachat-ios/Controller/AccountSwitchViewController.swift

```diff
@@ -67,6 +67,7 @@ class AccountSwitchViewController: UITableViewController {
         tableView.register(AccountCell.self, forCellReuseIdentifier: AccountCell.reuseIdentifier)
         tableView.rowHeight = AccountCell.cellHeight
         tableView.separatorStyle = .singleLine
+        tableView.bounces = false
```

Comment:
Since in most of the cases the user has not more than 3 accounts, the missing overscroll effect is not reconizable . In contrast, the overscroll effect is somewhat weird with one ore two accounts. What we can do is to enable .bounces of there are more than 3 accounts.

Reply:
yip, maybe this is a compromise. otherwise i'd prefer the old, always bouncing.